classes/simplemap.php

```diff
@@ -573,7 +573,7 @@ function add_taxonomy_fields( $atts, $taxonomy ) {
 
 			// Show taxes filters if allowed.
 			$tax_search = '';
-			$tax_label  = apply_filters( $php_taxonomy . '-text', __( $tax_object->labels->singular_name . ': ' ), 'simplemap' );
+			$tax_label  = apply_filters( $php_taxonomy . '-text', __( $tax_object->labels->singular_name . ': ' ), 'SimpleMap' );
```

Comment:
__[code_sniffer]__
* Spaces must be used to indent lines; tabs are not allowed
* Space after opening parenthesis of function call prohibited
* Expected 0 spaces before closing bracket; 1 found
* Line exceeds 120 characters; contains 129 characters

<!-- Comment by [SideCI](https://sideci.com) -->